server/service/src/requisition/program_settings/map.rs

```diff
@@ -96,3 +100,52 @@ fn period_is_available(
 
     number_of_requisitions_in_this_period < order_type.max_order_per_period as i64
 }
+
+/// Reduce periods by MAX_NUMBER_OF_HISTORIC_PERIODS and MAX_NUMBER_OF_FUTURE_PERIODS
+/// and sort in ascending order
+fn reduce_and_sort_periods(periods: Vec<PeriodRow>) -> Vec<PeriodRow> {
+    let now = date_now();
+    let (mut historic, mut future): (Vec<PeriodRow>, Vec<PeriodRow>) =
+        periods.into_iter().partition(|p| p.start_date < now);
```

Comment:
strictly speaking, historic would refer to periods where the end date is < now. What the list is showing for me, is two historic periods, the current period and three futures. I was confused for a bit, because I didn't realise that the currently active period should be considered part of the 3 'historic' ones.

This isn't a request to change - more something to be aware of when we write docs - it would be helpful to have historic, current and future periods mentioned.

Reply:
I had the same confusion actually, added comment in code 3f82ec1b40476acbb47719fc9557c1571736f05b, but yeah we would probably need something extra in public docs. Also I think we would need to have a more involved filtering (i.e. dont' show historic periods that are x number of periods old)